ledger-service/http-json/src/test/scala/com/digitalasset/http/query/ValuePredicateTest.scala

```diff
@@ -232,42 +244,71 @@ class ValuePredicateTest
       }
     }
 
-    // we aren't running the SQL, just looking at it
-    implicit val sjd: dbbackend.SupportedJdbcDriver = dbbackend.SupportedJdbcDriver.Postgres
-
     val sqlWheres = {
-      import doobie.implicits._, sjd.queries.Implicits._
+      import doobie.implicits._, dbbackend.Queries.Implicits._
       Table(
-        ("query", "type", "sql"),
-        ("42", VA.int64, sql"payload = ${s"""{"$dummyFieldName":42}""".parseJson}::jsonb"),
+        ("query", "type", "postgresql", "oraclesql"),
+        (
+          "42",
+          VA.int64,
+          sql"payload = ${s"""{"$dummyFieldName":42}""".parseJson}::jsonb",
```

Comment:
I’m a bit confused by the use of `parseJson`. Are you just using that to canonicalize the format? If not, I don’t see what you gain over just writing the literal directly.

Reply:
I think, when you have JSON data in code, it is a lot clearer to write it in JSON format when feasible rather than to write constructor calls.

Reply:
agreed, what I’m wondering is why you write it as a JSON string literal, then call `parseJson` on it only to then convert it back to a string literal. The only thing I can come up with is canonicalization.

Reply:
> then convert it back to a string literal

This does not happen in this case.

Reply:
I’m confused, I thought you’re interpolating it back into the sql string?

Reply:
Nope 🙂

Reply:
Ohhh, I see :facepalm: